libvast/src/system/disk_monitor.cpp

```diff
@@ -91,14 +91,14 @@ disk_monitor(disk_monitor_actor::stateful_pointer<disk_monitor_state> self,
       // have finished or we encountered an error.
       auto shared_guard
         = make_shared_guard([=] { self->state.purging = false; });
-      std::error_code ec{};
-      const auto index_dir
-        = std::filesystem::directory_iterator(self->state.dbdir / "index", ec);
-      if (ec)
+      std::error_code code{};
```

Comment:
```suggestion
      std::error_code err{};
```

This is our go-to name for local error variables that doesn't conflict with types. In longer functions, seeing `code` without the type is harder to reason about than seeing `err`.

Not worth changing, but worth pointing out, perhaps for future occurrences.

Reply:
Meh, `err` doesn't convey whether it's code or category.